.gitignore

```diff
@@ -1 +1,5 @@
 node_modules/
+jmeter.log
```

Comment:
makes sense. To try out, you can work with @mhdawson  and setup the repo on benchmarking machine and run it using test job that @mhdawson created.